dev/schemas/index.md

```
@@ -0,0 +1,14 @@
+outline: false
+title: vitepress-theme-openapi
+---
+
+<script setup lang="ts">
+import { useData } from 'vitepress'
+import spec from '../../docs/public/openapi-schemas.json'
+
+const { isDark } = useData()
+</script>
+
+<OASpec :spec="spec" :isDark="isDark" />
```

Markdownlint

Element: OASpec
Inline HTML

(MD033, no-inline-html)